pkg/option/config.go

```diff
@@ -387,6 +387,20 @@ const (
 	CTMapEntriesGlobalTCPName    = "bpf-ct-global-tcp-max"
 	CTMapEntriesGlobalAnyName    = "bpf-ct-global-any-max"
 
+	// CTMapEntriesTimeout* name option and default value mappings
+	CTMapEntriesTimeoutSYNName       = "bpf-ct-timeout-regular-tcp-syn"
+	CTMapEntriesTimeoutSYNDefault    = "1m"
+	CTMapEntriesTimeoutFINName       = "bpf-ct-timeout-regular-tcp-fin"
+	CTMapEntriesTimeoutFINDefault    = "10s"
+	CTMapEntriesTimeoutTCPName       = "bpf-ct-timeout-regular-tcp"
+	CTMapEntriesTimeoutTCPDefault    = "6h"
+	CTMapEntriesTimeoutAnyName       = "bpf-ct-timeout-regular-any"
+	CTMapEntriesTimeoutAnyDefault    = "1m"
+	CTMapEntriesTimeoutSrvTCPName    = "bpf-ct-timeout-service-tcp"
```

Comment:
Nit: maybe `s/CTMapEntriesTimeoutSrvTCPName/CTMapEntriesTimeoutSVCTCPName/`, as in the cilium-agent codebase we abbreviate "service" with "svc"?

Reply:
Ok, I can do that.